app/views/monitoring.html

```diff
@@ -57,375 +57,418 @@ <h1>
                       <div ng-if="(pods | hashSize) > 0">The current filters are hiding all pods.</div>
                       <span ng-if="podsLoaded && (pods | hashSize) === 0">There are no pods in this project.</span>
                     </em>
                   </div>
                 </div>
-                <div class="list-group-item list-group-item-expandable"
-                     ng-repeat-start="pod in filteredPods track by (pod | uid)"
-                     ng-click="toggleItem($event, this, pod)"
-                     ng-class="{'expanded': expanded.pods[pod.metadata.name]}">
-                  <div class="list-view-pf-checkbox">
-                    <button class="sr-only">{{expanded.pods[pod.metadata.name] ? 'Collapse' : 'Expand'}}</button>
-                    <span ng-if="expanded.pods[pod.metadata.name]">
-                      <span class="fa fa-angle-down"></span>
-                    </span>
-                    <span ng-if="!expanded.pods[pod.metadata.name]">
-                      <span class="fa fa-angle-right"></span>
-                    </span>
-                  </div>
-
-                  <div class="list-view-pf-main-info">
-                    <div class="list-view-pf-body">
-                      <div class="list-view-pf-description">
-                        <div class="list-group-item-heading">
-                          <a ng-href="{{pod | navigateResourceURL}}"><span ng-bind-html="pod.metadata.name | highlightKeywords : filterKeywords"></span></a>
-                          <small>created <span am-time-ago="pod.metadata.creationTimestamp"></span></small>
-                        </div>
-                        <div class="list-group-item-text">
-                          <status-icon status="pod | podStatus" disable-animation></status-icon>
-                          {{pod | podStatus | humanizeReason}}
-                          <small ng-if="(pod | podStatus) === 'Running'" class="text-muted">
-                            &ndash;
-                            {{pod | numContainersReady}}/{{pod.spec.containers.length}} ready
-                          </small>
-                        </div>
-                      </div>
-                      <div class="list-view-pf-additional-info">
-                        <div class="list-view-pf-additional-info-item">
-                          <image-names pod-template="pod" pods="[pod]"></image-names>
-                        </div>
-                      </div>
+                <div class="list-pf-item"
+                     ng-repeat="pod in filteredPods track by (pod | uid)"
+                     ng-class="{'active': expanded.pods[pod.metadata.name]}">
+                  <div class="list-pf-container list-pf-container-thin"
+                       ng-click="toggleItem($event, this, pod)">
+                    <div class="list-pf-chevron">
+                      <button class="sr-only">{{expanded.pods[pod.metadata.name] ? 'Collapse' : 'Expand'}}</button>
```

Comment:
This works, but it would be good to use the pattern from the overview which lets you tab to the chevron and expand it using the keyboard:

https://github.com/openshift/origin-web-console/blob/master/app/views/overview/_list-row-chevron.html